apps/openmw/mwmechanics/creaturestats.hpp

```diff
@@ -33,19 +33,22 @@ namespace MWMechanics
         MagicEffects mMagicEffects;
         Stat<int> mAiSettings[4];
         AiSequence mAiSequence;
-        bool mDead;
-        bool mDied;
-        bool mMurdered;
+
+        bool mDead,
+        mDied,
+        mMurdered,
+        mTalkedTo,
+        mAlarmed,
+        mAttacked,
+        mAttackingOrSpell,
+        mKnockdown,
+        mKnockdownOneFrame,
+        mKnockdownOverOneFrame,
+        mHitRecovery,
+        mBlock,
+        mRecalcMagicka;
         int mFriendlyHits;
```

Comment:
Can we please not do that? One separate declaration per variable (as we had before).